DAO/MessageDAO.py

```diff
@@ -146,3 +151,16 @@ def insert_message(self, cid, uid, message, img):
         cursor.connection.commit()
 
         return message_id
+
+    def insert_reply(self, message, uid, mid, cid, img=None):
```

Comment:
```python 
def insert_reply(self, message, uid, mid, cid, img=None):
        cursor = self.get_cursor()
        rid = self.insert_message(cid, uid, message, img=img)
        query = 'INSERT INTO replies (replied_to, reply) values (%s, %s)'
        cursor.execute(query, (mid, rid))
        reply_id = cursor.fetchone()['mid']
        self.conn.commit()
        return reply_id
```

```insert_message``` method already checks if ```img``` is not None and adds image accordingly

Reply:
Uploaded